Fsd cosmic sim

diaza opened the pull request:
Since we merged fsd_cosmic_sim into develop in larnd-sim, should we do the same here?

cuddandr committed:
Enable truth reference saving for FSD

diaza committed:
Load geometry in the first stage for FSD

cuddandr committed:
Further update FSD yamls for light simulation (remove deconv step)

Remove the waveform deconvolution step for now as the FSD specific
inputs are not available.

Fix a typo in the channel map in `LightEventGeneratorMC.yaml`

Change a few remaining 2x2 values to FSD values (e.g. nchantpc)

mjkramer committed:
Switch to Andrew's pylandau branch

Addresses build issues with recent versions of numpy and cython

cuddandr committed:
Another update to light channel mapping for FSD

Original channel map modified based on examining event displays to
achieve something that visually looked okay.

This may be incorrect and need further tweaks, or there is some other
place that needs to be corrected to use the original uploaded map.

Additional corrections to FSD light yamls.

cuddandr committed:
Merge branch 'develop' into fsd_cosmic_sim

Accepted code/changes in develop for merge conflicts.

Commit:
Disabled chips and channels for the FSD simulation

cuddandr committed:
Merge remote-tracking branch 'origin/fsd_cosmic_sim' into fsd_cosmic_sim

diaza committed:
Have MC tile layout match data

diaza committed:
Swap light channels to match current runs

diaza committed:
Incorporate disabled channels/chips to FSD simulation

diaza committed:
Merge pull request #212 from DUNE/fsd_geometry_fix

Fsd geometry fix

cuddandr commented:
Yes, we should. This was previously held up by some merge conflicts the last time I looked at it. I pulled develop into fsd_cosmic_sim, and had not got around to going back the other way since I am a little afraid of somehow overwriting some configs with old values given the previous conflicts.

diaza committed:
Correct FSD op channel mapping

diaza committed:
Merge pull request #222 from DUNE/feature_LUTflipper_fsd

Correct FSD op channel mapping

cuddandr committed:
Add gain and voltages to FSD MC yaml

Add the updated gain and voltages based on data conditions for the FSD
in CalibHitBuilderMC.yaml

cuddandr committed:
Update several FSD light reco YAMLs to be compatible develop branch

Most changes were copying in the new settings and dataset names from
examples on the develop branch.

Removed 'local_maxima' option from the SiPM and Sum HitFinder YAMLs
since it was not present in the develop branch.

cuddandr committed:
Merge branch 'develop' into fsd_cosmic_sim
